### PR TITLE
ISAICP-4624: RSS Feed - Collections

### LINKED ISSUE #1467: provide RSS/ATOM feeds
On the old joinup page I could subscribe to RSS feeds to stay up-to-date. Would be great if this would be possible with the new joinup as well.

### PR DESCRIPTION
**Pull request by brummbar**
Fixes #1467 .

### CODE REVIEW

**Comment by pfrenssen** [comment]
The latest test failed due to a timeout. I have restarted the test.

File: web/modules/custom/joinup_search/joinup_search.views.inc [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+      // Mark the table key alias as used.
+      $alias_tracker[$table_key] = TRUE;
+
+      $entity_type_id = $datasource->getEntityTypeId();
+      if (!$entity_type_id) {
+        continue;
+      }
+
+      /** @var \Drupal\Core\Entity\EntityTypeInterface $entity_type */
+      $entity_type = \Drupal::service('entity_type.manager')->getDefinition($entity_type_id);
```
[the remark this answers is not in the document]
Comment (brummbar) [reply web/modules/custom/joinup_search/joinup_search.views.inc]: Oopsie.

File: tests/src/Context/RssContext.php [hunk 2]
```diff
@@ -0,0 +1,178 @@
...
+    $rss = new \DOMDocument();
+
+    if ($rss->loadXML($this->getSession()->getPage()->getContent()) === FALSE) {
+      throw new \Exception('The page does not contain a valid RSS feed.');
+    }
+
+    if ($rss->schemaValidate(__DIR__ . '/../../schemas/RSS20.xsd') === FALSE) {
+      $message = "The RSS feed does not comply with the RSS schema.\n";
+      foreach (libxml_get_errors() as $error) {
+        $message .= sprintf('Error at line %s, column %s: %s', $error->line, $error->column, $error->message);
```
[the remark this answers is not in the document]
Comment (brummbar) [reply tests/src/Context/RssContext.php]: If I'm not mistaken the message has an empty line at the end already.

**Comment by brummbar** [comment]
Regarding the collection title change, maybe we can add the url as link template so aliases are generated through subpathauto which would take care of those itself?
Regarding deleted items, the effort needed for this is too big and the rss standard itself doesn't have real support for this use case that we shouldn't spend time on it.
The refresh time was a hard requirement in the ticket, that's why I didn't insist more on the caching.

**Comment by brummbar** [comment]
Tests for commit `2eb6c56` should have failed. Re-run in a [separate branch](https://app.continuousphp.com/git-hub/ec-europa/joinup-dev/build/82083c8a-4d92-48a5-bda5-93c2a76e0a77/logs/fdbc7d37-6bdd-4d8f-8f92-c42d0ec46e8e) to ensure that they fail without commit `da9dbc3`.

**Pull request merged**